### DISCUSSION TITLE
Is it possible to implement the ability to reuse images generated during the process?

### DISCUSSION THREAD

**Discussion by fumitakayano** [v8hid/infinite-zoom-automatic1111-webui#43] [Ideas]
Thank you v8hid for making this interesting extension.
I am very interested in this extension and would like to suggest an upgrade.

I was testing the current version and was curious about the png file that is generated before the movie generation.
Would it be possible to implement an additional feature to regenerate the movie by reusing this?

If so, I think it would be possible to process the png file using LoopBack, etc. to generate the movie more as intended.

https://user-images.githubusercontent.com/51348259/233340959-d6eeae68-7389-4b7c-8d9b-4638bb357246.mp4
![image](https://user-images.githubusercontent.com/51348259/233338948-603c614d-5b2b-454e-846d-db630b35069c.png)

**Comment by GeorgLegato** [comment]
not so easy , how shoulde we blend from last  outpainted frame to an png provided. we struggle actually on an exit image, to get an SEamless transition from/to given png.
initial img was easy since we outpainting FROM. now we talk about outpainting into a new img

**Comment by Oncorporation** [comment]
I had a similar challenge and have updated a pull request in the develop branch. In this, I have added key frames and expanded the JSON format to include a file location instead of a prompt.

Here is an example:
https://user-images.githubusercontent.com/5302844/233852006-ec3ec849-063a-41c3-8240-dc8c4f27429f.mp4

here is an example of the new json format
[sweet-dreams-infinite-zoom-prompts12-TEST-inverted.json.txt](https://github.com/v8hid/infinite-zoom-automatic1111-webui/files/11304244/sweet-dreams-infinite-zoom-prompts12-TEST-inverted.json.txt)

**Reply by Oncorporation** [reply]
updated my main branch https://github.com/Oncorporation/infinite-zoom-automatic1111-webui

I also maintain an shader filters extension for OBS Studio, and I will look into adding some effects for blending.

[GeorgLegato](https://github.com/GeorgLegato) is already working on something like this.

**Reply by v8hid** [reply]
@Oncorporation come to discord server we can communicate better there and make something good out of it.
